codesign/lib/src/utils.dart

```diff
@@ -30,6 +30,18 @@ enum FileType {
   }
 }
 
+Future<String> symlink(String folderPath, ProcessManager processManager) async {
```

Comment:
Docs?

Reply:
Sure, added doc and corrected typo in the newest commit. Thanks for the catch!